BLSQ/iaso
# POLIO-1791: update front-end subactivities

Add fields added in POLIO-1790 in front-end form

Related JIRA tickets : POLIO-1791

## Self proofreading checklist


## Doc

Tell us where the doc can be found (docs folder, wiki, in the code...).

## Changes

Explain the changes that were made.

- Add lqas/im fields to list API response
- Add fields in SubActivityForm
- Update formik state with new fields
- Update TS type
- Add validation for LQAS/IM fields
- Add translations

## How to test

- Create a Subactivity
- LQAS/IM fields should be visible
- Add dates and save
- Open subactivity
- Saved values should be displayed
- Try to add wrong dates: end date before start date or LQAS/IM start datre before subactivity end date: fields should be in error

## Print screen / video


https://github.com/user-attachments/assets/ab2d27dc-0f6c-470f-8815-340d9667105d



## Notes
Merges in POLIO-1790

## Follow the Conventional Commits specification

The **merge message** of a pull request must follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.

This convention helps to automatically generate release notes.

Use lowercase for consistency.

[Example](https://github.com/BLSQ/iaso/commit/8b8d7d3064138c1e57878f17b4eb922516ab0112):

```
fix: empty instance pop up

Refs: IA-3665
```

Note that the Jira reference is preceded by a _line break_.

Both the line break and the Jira reference are entered in the _Add an optional extended description…_ field.
